Front-End-Coders-Mauritius/frontendmu-astro
# UI to navigate to Next/Prev meetup from a single meetup page

### Changes

- Two new buttons, `previous meetup` and `next meetup` have been created in `SingleMeetupPage`.
- On desktop, images from the related meetups are loaded and used randomly as background for the buttons.
- On mobile, a simple arrow and text has been added to minimize space usage.

### Before


### After 


### How to test
- Ensure the buttons are responsive and you can navigate to the next and previous meetups without any errors.

### Closes #37
